azconfig/client/src/test/java/com/azure/azconfig/AzConfigTest.java

```diff
@@ -2,22 +2,14 @@
 // Licensed under the MIT License.
 package com.azure.azconfig;
 
-import com.azure.azconfig.models.Key;
-import com.azure.azconfig.models.KeyLabelFilter;
-import com.azure.azconfig.models.KeyValueFilter;
-import com.azure.azconfig.models.KeyValue;
-import com.azure.azconfig.models.KeyValueListFilter;
-import com.azure.azconfig.models.Label;
-import com.azure.azconfig.models.RevisionFilter;
+import com.azure.azconfig.models.*;
```

Comment:
Chackstyle will complain about '*' imports, please use individual class imports.

Reply:
Yeah. I had to disable this rule, for some reason IntelliJ uses * after 5... so I set it to 50.